packages/ibm-products/src/components/Coachmark/Coachmark.tsx

```diff
@@ -407,6 +419,7 @@ Coachmark.propTypes = {
 
   /**
    * What kind or style of Coachmark to render.
+   * Note that, `stacked` coachmarks are always open, even if `isOpenByDefault=false`.
```

Comment:
one last change
this still contradicts from the example we see on stacked variant in storybook, and i suggest to remove it to avoid any confusion.
```suggestion
```